index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Weather Report</title>
+    <link href="../styles/index.css" rel="stylesheet" />
```

Comment:
Your imports appear to be off by a folder:

```suggestion
    <link href="./styles/index.css" rel="stylesheet" />
```